proj/vs2019/NAS2D.vcxproj

```diff
@@ -249,6 +249,7 @@
     <ClInclude Include="..\..\include\NAS2D\Renderer\RendererNull.h" />
     <ClInclude Include="..\..\include\NAS2D\Renderer\Color.h" />
     <ClInclude Include="..\..\include\NAS2D\Renderer\Point.h" />
+    <ClInclude Include="..\..\include\NAS2D\Renderer\Vector.h" />

```

Comment:
This shouldn't be in the Renderer folder. It's a math concept and would make more sense in a `Math` subfolder.

Reply:
Agreed, though I think we need to address that for all geometries at the same time, in a separate PR.

Reply:
Some PR has to make the Math folder, why not start with this one? It'll make adding more later much easier.

Reply:
There are inter-dependencies between the headers. Either relative includes need to be updated, which is messy, or they all move at once.

The `Vector` class is strongly related to the `Point` class and the `Rectangle` class. It should go where those are.

I would really like to move this, I just don't think this is the PR to do it in.

Reply:
'K. :)

Reply:
Opened issue #358 to track.